mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      }
+      @Article{Parkinson2012,
+      AUTHOR = {Parkinson, C. L. and Cavalieri, D. J.},
+      TITLE = {Antarctic sea ice variability and trends, 1979--2010},
+      JOURNAL = {The Cryosphere},
+      VOLUME = {6},
+      YEAR = {2012},
+      NUMBER = {4},
+      PAGES = {871--880},
+      URL = {https://www.the-cryosphere.net/6/871/2012/},
+      DOI = {10.5194/tc-6-871-2012}
+      }
+      @article{Zwally2002,
+      author = {H. Jay Zwally and Josefino C. Comiso and Claire L. Parkinson and Donald J. Cavalieri and Per Gloersen},
+      title = {Variability of Antarctic sea ice 1979–1998},
+      journal = {Journal of Geophysical Research: Oceans},
+      volume = {107},
+      number = {C5},
+      pages = {9-1-9-19},
+      keywords = {sea ice, Antarctic, climate, passive microwave, Southern Ocean},
+      doi = {10.1029/2000JC000733},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2000JC000733},
+      year = {2002}
+      }
+    </bibtex>
+    <dataUrls>
+      - https://neptune.gsfc.nasa.gov/uploads/files/NH_IceArea_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/NH_IceExt_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/SH_IceArea_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/SH_IceExt_Monthly_1978-2012.txt
+    </dataUrls>
+    <preprocessing>
+      (missing)
+    </preprocessing>
+    <tasks>
+      - timeSeriesSeaIceAreaVol
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      IceSat Ice Thickness
+    </name>
+    <component>
+      seaice
+    </component>
+    <description>
+      This data set provides measurements of sea ice freeboard and sea ice
+      thickness for the Arctic region. The data were derived from measurements
+      made by from the Ice, Cloud, and land Elevation Satellite (ICESat)
+      Geoscience Laser Altimeter System (GLAS) instrument, the Special Sensor
+      Microwave/Imager (SSM/I), and climatologies of snow and drift of ice.
+    </description>
+    <source>
+      [NASA: Arctic Sea Ice Freeboard and Thickness](http://nsidc.org/data/NSIDC-0393)
+    </source>
+    <releasePolicy>
+      NASA data are not copyrighted; however, when you publish our data or
+      results derived therefrom, we request that you include an acknowledgment
+      within the text of the publication and reference list.
+      [Data Citation and Acknowledgements]
+      (https://podaac.jpl.nasa.gov/CitingPODAAC)
+    </releasePolicy>
+    <references>
+      [Yi and Zwally (2009)](https://doi.org/10.5067/SXJVJ3A2XIZT)
+    </references>
+    <bibtex>
+      @misc{Yi2009,
+      author = {D. Yi and H. J. Zwally},
+      title = {Arctic Sea Ice Freeboard and Thickness, Version 1},
+      year = {2009},
+      doi = {10.5067/7Q8HCCWS4I0R},
+      url = {https://doi.org/10.5067/SXJVJ3A2XIZT}
+      }
+    </bibtex>
+    <dataUrls>
+      (missing, requires registration with NSIDC)
```

Comment:
@milenaveneziani, can you figure out the URL?

Reply:
hmm, I realize tracking future changes in the data sets will be a task in itself.. I now see different text files, which also contain ice area for specific subregions of the Arctic/SO. The last column is the total hemispheric average (as before).
Here are the URLs:
https://neptune.gsfc.nasa.gov/uploads/files/NH_IceArea_Monthly_1978-2012.txt
https://neptune.gsfc.nasa.gov/uploads/files/SH_IceArea_Monthly_1978-2012.txt